### PR TITLE
Add saturation adjustment for generated color palette

### PR DESCRIPTION
**Pull request by Inbal-Tish**

### CODE REVIEW

File: src/style/colors.js [hunk 1]
```diff
@@ -123,11 +123,43 @@ class Colors {
...
 
-    const sliced = tints.slice(0, 8);
-    return sliced;
+    let sliced = tints.slice(0, 8);
+    const adjusted = adjustSaturation(sliced, color);
+    return adjusted || sliced;
   });
 }
 
+function adjustSaturation(colors, color) {
```
Comment (ethanshar) [inline src/style/colors.js]: Let's add some test to cover these logic

Comment (Inbal-Tish) [reply src/style/colors.js]: Ok

File: src/style/colors.js [hunk 2]
```diff
@@ -123,11 +123,43 @@ class Colors {
...
+  let array;
+  const lightnessLevel = 80;
+  const saturationLevel = 60;
+  const hsl = Color(color).hsl();
+  const lightness = Math.round(hsl.color[2]);
+
+  if (lightness > lightnessLevel) {
+    const saturation = Math.round(hsl.color[1]);
+    if (saturation > saturationLevel) {
+      array = [];
```
Comment (ethanshar) [inline src/style/colors.js]: sorry, should have wrote it before
let's shorten this a little..
you can
`array = _.map(colors, e => e !== color ? addSaturation(e, saturationLevel) : e)`

Comment (Inbal-Tish) [reply src/style/colors.js]: Cool

**Pull request merged**